### ISSUE TITLE
Unify values.yaml file examples comments

### ISSUE THREAD

**Issue by perk-sumo** [SumoLogic/sumologic-kubernetes-collection#837]
Let's agree on convention for examples and unify this across whole [values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/values.yaml) file.

As for now we use both: `# key:` (with space) and `#key:` (without space).
Also sometimes we have whole code blocks which should be unified as well, eg.
```
# key1:
  # key2: alice
  # key3: bob
```
where it should be:
```
# key1:
#   key2: alice
#   key3: bob
```
or even:
```
#key1:
#  key2: alice
#  key3: bob
```
Related to:
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/835 https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/836

**Comment by sumo-drosiek** [comment]
Can we add section to the contribution doc, because there is more topics like that:
 - one vs two spaces before comment
 - should comment be one or the `#` signed depends on wjhat it represents

**Comment by sumo-drosiek** [comment]
I agree that 
```
# key1:
#   key2: alice
#   key3: bob
```
should be the correct version

**Comment by pmalek-sumo** [comment]
Isn't this done already @perk-sumo ?

**Comment by perk-sumo** [comment]
@pmalek-sumo no, it's still something to be agreed on and done - we'll need to fix it for 1.3 release

**Comment by perk-sumo** [comment]
> Most of the times you'll be using this with an IDE anyway
I don't agree with that statement, most of the time clients will be doing this in their console over SSH IMO.

**Comment by sumo-drosiek** [comment]
@perk-sumo In both cases `# ` seems to be better solution

**Comment by perk-sumo** [comment]
Not for me, no.
Given that it's a YAML file and additional space can brake things I don't like the idea of having to figure out the proper tabulation.

**Comment by sumo-drosiek** [comment]
> I don't like the idea of having to figure out the proper indentation.

That's why we need to stick to one. We can add the explanation on the top of the configuration files

**Comment by sumo-drosiek** [comment]
We should use linter and I doubt any linter recognizes intention of the comment

https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.comments_indentation
https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.comments

**Comment by pmalek-sumo** [comment]
Here's an approach to start using `yamllint`. We can tweak it's config as we see fit https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1173

**Comment by pmalek-sumo** [comment]
Since #1173 has already been merged and we've decided that [managing comment indentation is tricky with yamllint](https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1178#issuecomment-847055082) should we close this, propose a change in yamllint (in an [issue thread](https://github.com/adrienverge/yamllint/issues/375) that I've created for this purpose) or maybe should we rather create a custom linter to enforce this?
